### PR TITLE
Hotfix/get url

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

https://jira.itkdev.dk/browse/SUPP0RT-531

#### Description

After reindex the database with information about, which covers was upload the generateUrl function is called, which contains errors.

#### Screenshot of the result

No screenshot.

#### Checklist

- [x] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

No comments.

### CODE REVIEW

File: src/Command/GetRemoteUrlsCommand.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+/**
+ * Class GetRemoteUrlsCommand.
+ */
+class GetRemoteUrlsCommand extends Command
+{
+    private CoverRepository $coverRepository;
+    private CoverService $coverStoreService;
+    private EntityManagerInterface $entityManager;
+
+    protected static $defaultName = 'app:cover:remote';
```
Comment (turegjorup) [inline src/Command/GetRemoteUrlsCommand.php]: Maybe `app:cover:get-remote-urls`

File: src/Command/GetRemoteUrlsCommand.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    /**
+     * CleanUpCommand constructor.
```
Comment (turegjorup) [inline src/Command/GetRemoteUrlsCommand.php]: `GetRemoteUrlsCommand` constructor

File: src/Command/GetRemoteUrlsCommand.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $limit = $input->getOption('limit');
+
+        $query = $this->coverRepository->getNoRemoveUrl($limit);
```
Comment (turegjorup) [inline src/Command/GetRemoteUrlsCommand.php]: Typo in `getNoRemoveUrl` -> `getNoRemoteUrl`

File: src/Repository/CoverRepository.php [hunk 4]
```diff
@@ -38,4 +38,26 @@ public function getIsNotUploaded(int $limit = 0): Query
...
+    /**
+     * Find all covers that do not have remote url defined.
+     *
+     * @param int $limit
+     *   Limit the number of records
+     *
+     * @return Query
+     *   DQL query
+     */
+    public function getNoRemoveUrl(int $limit = 0): Query
```
Comment (turegjorup) [inline src/Repository/CoverRepository.php]: Typo in `getNoRemoveUrl` -> `getNoRemoteUrl`

**Review by turegjorup — CHANGES_REQUESTED** [review]
Minor fixes required

File: src/Command/GetRemoteUrlsCommand.php (hunk 1 above)
Comment (cableman) [reply src/Command/GetRemoteUrlsCommand.php]: Fixed

File: src/Command/GetRemoteUrlsCommand.php (hunk 2 above)
Comment (cableman) [reply src/Command/GetRemoteUrlsCommand.php]: Fixed

File: src/Command/GetRemoteUrlsCommand.php (hunk 3 above)
Comment (cableman) [reply src/Command/GetRemoteUrlsCommand.php]: Fixed

File: src/Repository/CoverRepository.php (hunk 4 above)
Comment (cableman) [reply src/Repository/CoverRepository.php]: Fixed

**Review by turegjorup — APPROVED** [review]

**Pull request merged**